djangocms_text_ckeditor/tests/test_widget.py

```diff
@@ -36,7 +38,6 @@ def test_sub_plugin_config(self):
     def test_plugin_edit(self):
         page = create_page(title='pagina', template='page.html', language='en')
         add_plugin(page.get_placeholders('en').get(slot='content'), 'TextPlugin', 'en', body="Lorem ipsum")
-        page.publish('en')
```

Comment:
Can you add a description to the PR intro that states that publishing has been removed from the core of Django CMS v4?

Reply:
Done.